hns/hns_test.go

```diff
@@ -26,6 +30,8 @@ func TestHNS(t *testing.T) {
 var _ = BeforeSuite(func() {
 	err := common.HardResetHNS()
 	Expect(err).ToNot(HaveOccurred())
+	err = common.HardResetHNS()
```

Comment:
This is leftover from that weird bug where every second test would fail. This seems to be fixed now, so this is dead code.